measuring-client/src/main/java/de/cyface/app/ui/LoginActivity.java

```diff
@@ -275,10 +259,10 @@ private void setServerUrl() {
     }
 
     private void registerFeedbackLink() {
-        final Intent emailIntent = generateFeedbackEmailIntent(this, getString(R.string.your_request), SUPPORT_EMAIL);
-        final TextView feedbackLink = findViewById(R.id.login_link_contact_us);
-        feedbackLink.setOnClickListener(
-                v -> startActivity(Intent.createChooser(emailIntent, getString(R.string.feedback_choose_email_app))));
+        final var browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://app.cyface.de/registration"));
+        final var registrationLink = findViewById(R.id.login_link_registration);
+        registrationLink.setOnClickListener(
+                v -> startActivity(Intent.createChooser(browserIntent, getString(R.string.feedback_choose_email_app))));
```

Comment:
What does registration have to do with Feedback?

Reply:
Good Catch. I'll fix it, thanks